src/calendar/index.tsx

```diff
@@ -301,9 +298,6 @@ Calendar.propTypes = {
   ...CalendarHeader.propTypes,
   ...Day.propTypes,
   theme: PropTypes.object,
-  firstDay: PropTypes.number,
-  displayLoadingIndicator: PropTypes.bool,
-  showWeekNumbers: PropTypes.bool,
```

Comment:
Why did we remove this?

Reply:
`showWeekNumbers` is a CalendarHeader's prop, and CalendarProps inherits from it so it gets it from there and there's no need to redefine it

Reply:
I'm not sure if they're set automatically somehow but I don't see that `CalendarHeader` even have `propTypes` set anywhere.

Reply:
oh you mean the prop type... well, this should be removed all over but there was an issue there that I need to confirm first